pkg/asset/manifests/openshift.go

```diff
@@ -45,6 +47,12 @@ type Openshift struct {
 	FileList []*asset.File
 }
 
+// Used for templating the baremetal CR.
+type baremetalTemplateData struct {
+	Baremetal     *baremetal.Platform
+	ProvisioningOSDownloadURL string
```

Comment:
These should be aligned to make go fmt happy

Reply:
Doh!  I thought for sure I ran gofmt on it.  Thanks.